PointOfDispensingApp/config.js

```diff
@@ -4,5 +4,16 @@ export default {
         UNDETERMINED: 0,
         VACCINATION_PROCEED_YES: 1,
         VACCINATION_PROCEED_NO: 2   
-    }	
+    },	
+
+    patientWorkflowState:
+    {
+        INITIAL: 0,
```

Comment:
We should come up with a better name for this. Initial doesn't tell me where they're at

Reply:
It's really just the default/initial state before the patient name is looked up.  Do you have a better suggestion?

Reply:
I'd try `noPatientLoaded` since the state is basically you have a patient or there's no patient

Reply:
this too^

Reply:
Okay - I changed this to NO_PATIENT_LOADED